formapp/templates/usermanual.html

```diff
@@ -221,5 +221,57 @@ <h2>Other Documents</h2>
+</div>
+<u><h2>Formbar</h2></u>
+<div>
+    <h3>Background Music and Sound Effects</h3>
+    <p>Both of these tabs display a list of sounds you can play. Background music is a collection songs added by students or teachers overtime. Sound effects is a collection of non-looping sounds you can play.</p>
+    <h3>Advanced Homepage/Advanced Mode</h3>
+    <p>Not reccomended for those who don't understand the formbar well, it'll allow you to do everything the normal formbar can in a much more compact interface. It's a lot more technical than the normal layout and, once again, not reccomended for new users.</p>
+    <h3>Set Default Homepage</h3>
+    <p>This will allow you to change the type of page you user, the default, advanced, or quick pages are dependant on user preference, we reccomend sticking with default until you understand the formbar completely.</p>
+    <h3>VirtualBar</h3>
+    <p>The virtualbar will allow you to view a visual list of all poll answers, it will not display names.</p>
+</div>
+<u><h2>Games</h2></u>
+<div>
+    <p>The game section is filled with fun little mini-games you can play on your downtime. These were all made from scratch and some even include fun trivia about the class!</p>
+    <h3>Leaderboards</h3>
+    <p>The Leaderboard displays all the scores for each game and ranks them from best to worst.</p>
+</div>
+<u><h2>Teacher</h2></u>
+<div>
+    <h3>Teacher Controls</h3>
+    <p>The control panel is where teachers can view a list of all students, change user permissions, manage user profiles, view poll history and help/bathroom tickets, and full reboot the formbar. </p>
+    <h3>User Accounts</h3>
+    <p>This will allow teachers to change a students password per their request, delete their accounts, and reward them with digipogs. You just have to click the corresponding link and it'll guide you from there.</p>
+    <h4>Changing Passwords</h4>
+    <p>To change a user's password, after the input box comes up, you will enter the desired new password and hit "OK" have the student login once again.</p>
+    <h4>Adding and Removing Digipogs</h4>
+    <p>To add digipogs simply enter a static whole number and hit "OK" to remove them put a minus sign before the number to make a negative and then hit "OK".</p>
+    <h4>Lesson Creation</h4>
+    <p>Lessons are what you will use to teach each and every student in your class. You will utilize your controls, the lightbar, and more in each lesson. Creating a lesson is simple and easy.
+        Process and starting a new excel spreadsheet: Go into your excel in your computer, Start a blank excel sheet, Make two columns with word and definition respectively, Make the words you need in the word column, and Put the definition for those words in the definition column.
+        Uploading a lesson: Go to the Teacher tab in formbar, Go to the red lesson tab in the teacher tab, Click upload a lesson, Choose the excel spreadsheet file, and Confirm and upload.
+    </p>
```

Comment:
Creating Lessons is MUCH more complicated than this. Please actually use the lesson feature, and look at all of the sheets in the example data.